src/modules/clark/learning-object-module/changelog.routes.ts

```diff
@@ -0,0 +1,15 @@
+import { HTTPMethod } from "../../../shared/types/http-method.type";
+import { ProxyRoute } from "../../../shared/types/proxy-route.type";
+
+export const CHANGELOG_ROUTES: ProxyRoute[] = [
+    {
+        method: HTTPMethod.GET,
+        path: "/learning-objects/:cuid/changelogs",
+        auth: true,
+    },
+    {
+        method: HTTPMethod.POST,
+        path: "/learning-objects/:cuid/changelog",
```

Comment:
Is this supposed to be changelog or changelogs? Just wondering based on the structure from the route above.